### PR TITLE
F t34084 add missing hooks selectors

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T34084

- Add missing Hooks/Selectors.

### CODE REVIEW

File: templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig [hunk 1]
```diff
@@ -1270,10 +1274,13 @@
                     class="btn btn--primary"
                     id="saveConfig"
                     name="saveConfig"
-                    data-cy="saveConfig"
+                    data-cy="administrationEdit:saveConfig"
```
Comment (spiess-demos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig:1277]: no section = no prefix, i'd say.

```suggestion
                    data-cy="saveConfig"
```

File: templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig [hunk 2]
```diff
@@ -1270,10 +1274,13 @@
...
                     type="submit"
                     value="{{ "save"|trans }}">
-                <a class="btn btn--secondary" href="{{ path('DemosPlan_procedure_administration_get') }}">
+                <a
+                    class="btn btn--secondary"
+                    data-cy="administrationEdit:abort"
```
Comment (spiess-demos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig:1282]:
```suggestion
                    data-cy="abortConfig"
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/basicSettings/ExportSettings.vue [hunk 3]
```diff
@@ -30,25 +31,29 @@
       <dp-checkbox-group
         aria-labelledby="submitter"
         :options="submitterFields"
+        data-cy="submitterFields:"
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/procedure/basicSettings/ExportSettings.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/2815/commits/245c7974c7981fc280268bd27fcf47eef3796539

File: client/js/components/procedure/basicSettings/ExportSettings.vue [hunk 4]
```diff
@@ -30,25 +31,29 @@
...
         @update="checked => updateCheckedFields(checked)"
         :selected-options="getSelectedOptions(submitterFields)" />
     </div><!--
  --><dp-checkbox-group
       :label="Translator.trans('statement.data')"
       :options="metaDataFields"
+      data-cy="metaDataFields:"
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/procedure/basicSettings/ExportSettings.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/2815/commits/245c7974c7981fc280268bd27fcf47eef3796539

File: client/js/components/procedure/basicSettings/ExportSettings.vue [hunk 5]
```diff
@@ -30,25 +31,29 @@
...
       @update="checked => updateCheckedFields(checked)"
       :selected-options="getSelectedOptions(metaDataFields)"
     class="inline-block align-top u-1-of-4-wide u-1-of-2-desk u-1-of-2-lap u-1-of-1-palm u-mb u-pr-2" /><!--
  --><dp-checkbox-group
       v-if="hasPermission('field_procedure_elements')"
       :label="Translator.trans('documents')"
       :options="documentFields"
+      data-cy="documentFields:"
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/procedure/basicSettings/ExportSettings.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/2815/commits/245c7974c7981fc280268bd27fcf47eef3796539

File: client/js/components/procedure/basicSettings/ExportSettings.vue [hunk 6]
```diff
@@ -30,25 +31,29 @@
...
       @update="checked => updateCheckedFields(checked)"
       :selected-options="getSelectedOptions(documentFields)"
     class="inline-block align-top u-1-of-4-wide u-1-of-2-desk u-1-of-2-lap u-1-of-1-palm u-mb" /><!--
  --><dp-checkbox-group
       :label="Translator.trans('publication')"
       :options="publicationField"
+      data-cy="publicationField:"
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/procedure/basicSettings/ExportSettings.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/2815/commits/245c7974c7981fc280268bd27fcf47eef3796539

File: templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig [hunk 7]
```diff
@@ -166,7 +166,7 @@
                                     <dp-multiselect
                                         id="r_agency"
                                         v-model="selectedAgencies"
-                                        data-cy="selectedAgencies"
+                                        data-cy="administrationEdit:selectedAgencies"
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig]: Thanks, done in https://github.com/demos-europe/demosplan-core/pull/2815/commits/2c8a4022ac4416de44d96a04845c1b9836c7bfeb

**Review by spiess-demos — APPROVED** [review]
lgtm (code review)

**Pull request merged**